src/gatsby/src/pages/active-advisories.js

```diff
@@ -129,6 +129,26 @@ const PublicActiveAdvisoriesPage = ({ data }) => {
     getType: getAdvisoryType,
   }
 
+  const compareAdvisories = (a, b) => {
+    const getDate = (advisory) => {
+      if (advisory.isAdvisoryDateDisplayed) {
+        return new Date(advisory.advisoryDate)
+      }
+      if (advisory.isEffectiveDateDisplayed) {
+        return new Date(advisory.effectiveDate)
+      }
+      if (advisory.isUpdatedDateDisplayed) {
+        return new Date(advisory.updatedDate)
+      }
+      // Default to epoch if no date is displayed
+      return new Date(0)
+    }
+    const dateA = getDate(a)
+    const dateB = getDate(b)
+    // Sort in descending order (latest date first)
+    return dateB - dateA
+  }
```

Comment:
This works well the way it is, but if you wanna tweak it a bit, I had a thought:
`getDate` doesn't need to be re-defined every time `compareAdvisories` does a comparison, since `getDate` is a pure function. You can move the `getDate` definition outside of `compareAdvisories` (maybe call it `getAdvisoryDate` or something for clarity, though).

I think it will make no difference performance-wise, but it's just a suggestion!